tests/posix/common/CMakeLists.txt

```diff
@@ -4,6 +4,6 @@ cmake_minimum_required(VERSION 3.13.1)
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(posix_common)
 
-target_include_directories(app PRIVATE $ENV{ZEPHYR_BASE}/include/posix)
```

Comment:
Code should not be commented out.

Reply:
Fixed, and also removed this directive from the other posix test.